heasarc/heasarc/Dockerfile

```diff
@@ -176,3 +176,36 @@ RUN printf "\n\
 if [ -f /home/${user}/workspace/Storage/*/persistent/.bashrc ]; then\n\
     . /home/${user}/workspace/Storage/*/persistent/.bashrc\n\
 fi\n" >> /home/${user}/.bashrc
+
+## add js9 ##
+RUN mamba install -y -c conda-forge -n $pythonenv \
+    cfitsio \
+    sidecar \
+    jupyter-server-proxy \
+    # fix for socket.io to work
+    && sed -i "s/subprotocols=self.subprotocols,/#&/" \
+        /home/$user/miniconda3/envs/heasoft/lib/python3.10/site-packages/jupyter_server_proxy/handlers.py \
+    && mamba clean -y --all
+
+ENV JS9_WEB_PATH=/opt/js9-web/
+USER root
+RUN git clone -b jh_updates --depth=1 https://github.com/zoghbi-a/js9.git /tmp/js9 && \
```

Comment:
This is nitpicky, but the other RUN commands in the Dockerfile have the `&&` at the beginning of the continued lines, but this RUN command has the `&&` at the end of the continued lines. Consistency is the hobgoblins of little minds, of course.... 😄